package.json

```diff
@@ -1,34 +1,37 @@
 {
-	"name": "project_grads",
-	"version": "1.0.0",
-	"description": "",
-	"main": "index.js",
-	"scripts": {
-		"test": "npm run test:client && npm run test:server",
-		"test:client": "docker-compose -f dev-docker-compose.yml run -e COVERAGE_THRESHOLD=80 client bash ./travis_tests.sh",
-		"test:server": "docker-compose -f dev-docker-compose.yml run -e COVERAGE_THRESHOLD=40 server bash ./travis_tests.sh",
-		"dockerup": "docker-compose -f dev-docker-compose.yml up --build",
-		"dockerdown": "docker-compose -f dev-docker-compose.yml down --rmi all"
-	},
-	"repository": {
-		"type": "git",
-		"url": "git+https://github.com/FACxBeamery/grads-final-project.git"
-	},
-	"keywords": [],
-	"author": "",
-	"license": "ISC",
-	"bugs": {
-		"url": "https://github.com/FACxBeamery/grads-final-project/issues"
-	},
-	"homepage": "https://github.com/FACxBeamery/grads-final-project#readme",
-	"dependencies": {
-		"airbnb": "0.0.2",
-		"eslint-config-airbnb": "^18.0.1",
-		"eslint-config-prettier": "^6.7.0",
-		"mongo": "^0.1.0"
-	},
-	"devDependencies": {
-		"enzyme": "^3.10.0",
-		"enzyme-adapter-react-16": "^1.15.1"
-	}
+    "name": "project_grads",
+    "version": "1.0.0",
+    "description": "",
+    "main": "index.js",
+    "scripts": {
+        "test": "npm run test:client && npm run test:server",
+        "test:client": "docker-compose -f dev-docker-compose.yml run -e COVERAGE_THRESHOLD=80 client bash ./travis_tests.sh",
+        "test:server": "docker-compose -f dev-docker-compose.yml run -e COVERAGE_THRESHOLD=40 server bash ./travis_tests.sh",
+        "dockerup": "docker-compose -f dev-docker-compose.yml up --build",
+        "dockerdown": "docker-compose -f dev-docker-compose.yml down --rmi all"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/FACxBeamery/grads-final-project.git"
+    },
+    "keywords": [],
+    "author": "",
+    "license": "ISC",
+    "bugs": {
+        "url": "https://github.com/FACxBeamery/grads-final-project/issues"
+    },
+    "homepage": "https://github.com/FACxBeamery/grads-final-project#readme",
+    "dependencies": {
+        "@hapi/joi": "^16.1.8",
+        "airbnb": "0.0.2",
+        "eslint-config-airbnb": "^18.0.1",
+        "eslint-config-prettier": "^6.7.0",
+        "joi": "^14.3.1",
```

Comment:
we don't want to be using `joi` since it's deprecated in favour of `@hapi/joi`. This might affect how you've written your serverside validation

Reply:
Never used joi, not sure where it has come from!